src/components/Home/Hero/HeroSection.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { ButtonHTMLAttributes } from 'react'
+import cn from 'classnames'
+
+import { ReactComponent as ArrowRight } from '../../../../static/img/arrow-right-black.svg'
+import { ReactComponent as ArrowDown } from '../../../../static/img/arrow-down-white.svg'
+
+import * as styles from './styles.module.css'
+import shortenNumber from '../../../utils/format'
+import useStars from '../../../gatsby/hooks/stars'
+import Link from '@dvcorg/gatsby-theme-iterative/src/components/Link'
+import { navigate } from 'gatsby'
+import { StaticImage } from 'gatsby-plugin-image'
+
+interface ISectionProps {
+  className?: string
+  children?: React.ReactNode
+}
+
+const SectionWrapper = ({
+  className,
+  children,
+  maxWidth = 'lg'
+}: ISectionProps & {
+  maxWidth?: 'lg' | 'xl' | '2xl'
+}) => (
+  <div className={cn('w-full flex relative', className)}>
+    <div className="absolute w-full h-full flex flex-row">
+      <div className="w-1/2 h-full bg-dark" />
+      <div className="w-1/2 h-full bg-light" />
+    </div>
+    <div
+      className={cn('w-full z-10 flex m-auto', {
+        'max-w-screen-lg': maxWidth === 'lg',
+        'max-w-screen-xl': maxWidth === 'xl',
+        'max-w-screen-2xl': maxWidth === '2xl'
+      })}
+    >
+      {children}
+    </div>
+  </div>
+)
+
+const Section = ({ className, children }: ISectionProps) => (
+  <div
+    className={cn(
+      'w-1/2 px-6 py-10',
+      'flex flex-col items-center justify-center gap-8',
+      'text-center',
+      className
+    )}
+  >
+    {children}
+  </div>
+)
+
+const Badge = ({ className, children }: ISectionProps) => (
+  <div
+    className={cn(
+      'px-1.5 md:px-3 py-0 rounded-lg',
+      'w-fit',
+      'mx-auto',
+      'flex items-center justify-center gap-2',
+      'text-xl font-medium',
+      className
+    )}
+  >
+    {children}
+  </div>
+)
+
+export const CTAButton = ({
+  className,
+  children,
+  ...props
+}: ISectionProps & ButtonHTMLAttributes<HTMLButtonElement>) => (
+  <button
+    className={cn(
+      'rounded-lg',
+      'py-1 px-2',
+      'md:py-4 md:px-5',
+      'text-base font-medium ',
+      'md:text-xl',
+      'flex items-center',
+      className
+    )}
+    {...props}
+  >
+    {children}
+  </button>
+)
+
+const HeroSection = () => {
+  const stars = useStars()
+
+  return (
+    <>
+      {/* Title Section */}
```

Comment:
@djsauble @yathomasi  - I have some HL feedback about the design of the hero section (black/white halves)


Even though I already saw the design before and knew what to expect, when I first loaded the page it "looked broken", why?

- the hero section is cut even on larger screens until you scroll to it
- no horizontal borders

This means my mind initially "ignored" the white DVC half (right) and it looked as if the page was the left DVCx part, with black background, and it was somehow offset and overflowed to the left.

I understand the original design but this first impression is a bit jarring.

Maybe allowing borders (bg gradient-colour) on the sides (not letting the white/black section fill the entire width) would help center the eyes?

wdyt @djsauble ?

<details>
  <summary>this is how it looked like on a 24'' screen</summary>
  
  ![Screenshot 2023-08-18 at 11 58 59](https://github.com/iterative/dvc.org/assets/18181802/9de35641-2974-4c73-a9a5-cb582aaa003b)

</details>

Reply:
I agree with you. We're trying to not change the design of the page more than necessary, but I think this would benefit from some tweaks.

This is what it looks like with a max-width and even a drop-shadow to add some continuity to the two sides of the section. wdyt? cc @yathomasi 

<img width="1848" alt="Proposed design (Desktop)" src="https://github.com/iterative/dvc.org/assets/186715/141ba136-c81e-4a6b-b0b7-f6c7833a912c">

Reply:
Thanks @djsauble  - it looks better to me personally, eliminating the effect which bothered me a bit!

Reply:
On discussing this with @volkfox, since the new dvc.ai splash screen uses a similar black/white background motif, we want to avoid creating the same effect on this page. Instead, we'll remove the solid background color behind the diagram.

https://www.figma.com/file/nq7SOejyDtiaoxk4FplvyO/DQL?type=design&node-id=2486%3A2515&mode=design&t=ymxljJn77adO69ER-1

Reply:
https://github.com/iterative/dvc.org/pull/4803